cpp/src/arrow/pretty_print_test.cc

```diff
@@ -772,6 +772,11 @@ TEST_F(TestPrettyPrint, BinaryNoNewlines) {
   options.window = 2;
   expected = "[666F6F,626172,...,,FF]";
   CheckPrimitive<BinaryType, std::string>(options, is_valid, values, expected, false);
+
+  // With truncated element size
+  options.element_size_limit = 1;
+  expected = "[6+5,6+5,...,,F+1]";
```

Comment:
The output is a bit confusing. How about something like `6 (...)` or `6 (... 5 chars omitted)` instead of `6+5`?
(of course, with such a small element size it seems a bit ridiculous to append that much, but I suspect that will not be a common case)